### PR TITLE
Converted App into Vuetify and initiated the main layout

### PR DESCRIPTION
**Pull request by gayanSandamal**

### CODE REVIEW

File: src/pages/index.vue [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+const isEdit = ref(false);
+
+const tagColours = computed(() => ["indigo", "green", "red", "blue", "orange"]);
+
+const onClear = () => {
+  newItem.value = { title: "", brandId: null, edit: false };
+};
+const onEditEnable = (item) => {
+  newItem.value = { ...item, brandId: item.brandId, edit: true };
+  console.log('new item: ', newItem.value)
```
Comment (sanuja-ariyapperuma) [inline src/pages/index.vue]: Remove this console log

File: src/pages/index.vue [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+const newItem = ref({
+  title: "",
+  brandId: null,
+  edit: false,
+  isActive: false,
+  categories: [],
+});
+
+const filteredList = computed(() =>
+  // eslint-disable-next-line no-extra-boolean-cast
```
Comment (sanuja-ariyapperuma) [inline src/pages/index.vue:179]: What is the reason for disabling eslint? 
Is it necessary?

**Review by sanuja-ariyapperuma — CHANGES_REQUESTED** [review]
PTAL

File: src/pages/index.vue (hunk 1 above)
Comment (gayanSandamal) [reply src/pages/index.vue]: Removed

File: src/pages/index.vue (hunk 2 above)
Comment (gayanSandamal) [reply src/pages/index.vue:179]: !!! (triple negation) shows eslint warnings but it's a shorthand for checking not empty strings

**Review by sanuja-ariyapperuma — APPROVED** [review]
LGTM

**Pull request merged**